doc/workflow/workflow.rst

```diff
@@ -57,7 +57,7 @@ of the main regression equation and the propensity score.
 
 In step 1., we initialize the data-backend and thereby declare the role of the outcome, the treatment and the confounding variables.
 
-We use data from the 1991 Survey of Income and Program Participation which is available via the function fetch_401k [link to docu].
+We use data from the 1991 Survey of Income and Program Participation which is available via the function `fetch_401k <https://docs.doubleml.org/stable/api/generated/doubleml.datasets.fetch_401K.html>`_.
```

Comment:
Link both R and Python docu? Additionally the function names are unfortunately not aligned between R and Python.